openedx/core/djangoapps/content/block_structure/models.py

```diff
@@ -172,19 +234,15 @@ def _prune_files(cls, data_usage_key, num_to_keep=None):
             files_to_delete = all_files_by_date[:-num_to_keep] if num_to_keep > 0 else all_files_by_date
             cls._delete_files(files_to_delete)
             log.info(
-                'BlockStructure: Deleted %d out of total %d files in store; data_usage_key: %r, num_to_keep: %d.',
+                u'BlockStructure: Deleted %d out of total %d files in store; data_usage_key: %s, num_to_keep: %d.',
                 len(files_to_delete),
                 len(all_files_by_date),
                 data_usage_key,
                 num_to_keep,
             )
 
-        except Exception as error:  # pylint: disable=broad-except
-            log.exception(
-                'BlockStructure: Exception when deleting old files; data_usage_key: %r, %r',
-                data_usage_key,
-                error,
-            )
+        except Exception:  # pylint: disable=broad-except
+            log.exception(u'BlockStructure: Exception when deleting old files; data_usage_key: %s.', data_usage_key)
```

Comment:
**Note to reviewers:** No reason to explicitly log the `error` when using `log.exception`.  So I updated the code.